01_fib/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int) {
+
```

Comment:
Remove blank lines add beginning/end of block.
Could be further simplified to:
```go
	a, b := 1, 1
	for i := 1; i <= n; i++ {
		fmt.Fprintln(out, a)
		a, b = b, a+b
	}
```

Reply:
Updated the logic to print Negafibonacci numbers for negative `n`. And addressed your comments. Please review.